src/static_types.rs

```diff
@@ -99,6 +99,9 @@ pub mod westend {
 		>(16)
 	);
 
+	// SYNC https://github.com/paritytech/polkadot-sdk/blob/master/polkadot/runtime/westend/src/lib.rs#L451-#L453.
```

Comment:
A crazy idea would be to have a macro which keeps this in sync at compile time or a CI job that fetches that value, but it sounds a bit too complicated since the signed phase depends on multiple constants defined in substrate 🤔